src/modules/site-v2/templates/tools/phenotype_database/report.html

```diff
@@ -38,20 +38,27 @@
         <div class="col col-md-6 col-xl-3 text-bg-light mb-5 p-5 rounded-2 shadow-sm">
             <h2 class="h3 text-center mb-5">Trait Information</h2>
 
+            {%- if data.num_traits > 1 %}
```

Comment:
do we need a 'nice' fallback message in case something went wrong and num_traits === 0?